jni/src/test/java/com/horizen/schnorrnative/SchnorrKeyPairTest.java

```diff
@@ -1,93 +1,93 @@
-            -82, 105, 29, 75, 126, -62, -57, -26, 21, 22, 98, -106, 19, 58, 15, -12, -6, 123, -125, -41, -4, 82, -102,
-            -33, 100, -57, 92, -23, 114, -118, 83, -52, -8, 79, 11, 10, 24, 83, -74, 125, -27, -114, -29, -31, 82, -85,
-            126, 117, 79, -68, 49, 79, 18, -50, 31, -28, -99, 75, -74, 108, -6, 0, 17, 0, 32, -83, 112, 35, -51, -71, 0,
-            0, -65, -74, -110, 117, 23, -70, -105, -13, 35, -75, -118, 67, 23, 126, 40, -89, -21, 53, -63, 79, -41, -8,
-            -98, 62, -41, -12, 123, 79, 94, 61, 87, 122, -101, -47, 119, 86, 36, -57, -100, 101, 77, -92, 55, -41, -26,
-            77, -29, 5, 46, -62, -122, 110, 83, -19, -58, -37, 13, 96, 64, 46, 56, 20, 69, 12, -95, -113, -123, 118, 93,
-            -22, 13, -91, 117, -100, -22, -103, -81, -118, -33, -76, -18, -100, -119, 11, -37, 12, -46, 28, -43, -105,
-            66, -117, -100, 39, 0, 0
-        };
-
-        try
-        (
-            SchnorrSecretKey sk = SchnorrSecretKey.deserialize(skBytes);
-            SchnorrKeyPair keyPair = new SchnorrKeyPair(sk);
-            FieldElement message = FieldElement.deserialize(messageBytes);
-            SchnorrSignature sig = SchnorrSignature.deserialize(sigBytes)
-        )
-        {
-            assertNotNull("sk deserialization must not fail", sk);
-            assertNotNull("message deserialization must not fail", message);
-            assertTrue("Signature must be verified", keyPair.getPublicKey().verifySignature(sig, message));
-        }
-
-    }
-
-    @Test
-    public void testRandomSignVerify() throws Exception {
-
-        int samples = 100;
-
-        for (int i = 0; i < samples; i++) {
-            try
-            (
-                SchnorrKeyPair keyPair = SchnorrKeyPair.generate();
-                FieldElement fieldElement = FieldElement.createRandom();
-                FieldElement wrongFieldElement = FieldElement.createRandom()
-            )
-            {
-                assertNotNull("Key pair generation was unsuccessful.", keyPair);
-                assertTrue("Public key verification failed.", keyPair.getPublicKey().verifyKey());
-
-                try(SchnorrSignature signature = keyPair.signMessage(fieldElement))
-                {
-                    assertNotNull("Attempt to sign message failed.", signature);
-                    assertTrue("Signature must be verified", keyPair.getPublicKey().verifySignature(signature, fieldElement));
-                    assertFalse("Signature must not be verified", keyPair.getPublicKey().verifySignature(signature, wrongFieldElement));
-                }
-            }
-        }
-    }
-}
+//package com.horizen.schnorrnative;
+//
+//import com.horizen.librustsidechains.FieldElement;
+//import org.junit.Test;
+//
+//import java.util.Arrays;
+//
+//import static org.junit.Assert.*;
+//
+//public class SchnorrKeyPairTest {
+//
+//    @Test
+//    public void testGenerate() throws Exception {
```

Comment:
Don't see any usage of hardcoded data here. Maybe we can keep it ?

Reply:
Yes, this test is working, I've uncommented it